From 6f26875eba82fa205a39c3c6a3e577a2a0e42606 Mon Sep 17 00:00:00 2001
From: Andrew Siplas <andrew@asiplas.net>
Date: Sun, 18 Jul 2021 13:54:42 -0400
Subject: [PATCH] Change ambiguous `python` to `python3`

Fedora's RPM linting warns of ambiguous python shebang in
`/usr/bin/m64py`.

Changed to `#!/usr/bin/python3`.

Fixes #186
---
 bin/m64py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/bin/m64py b/bin/m64py
index bdd5911..3519124 100755
--- a/bin/m64py
+++ b/bin/m64py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python3
 # -*- coding: utf-8 -*-
 
 # Author: Milan Nikolic <gen2brain@gmail.com>
